File: api/Bussiness/Services/Stores/Dtos/StoreDto.cs
namespace Bussiness.Services.Stores.Dtos
{
    public class StoreDto
    {
        public int Id { get; set; }
        public string? StoreName { get; set; }
        public TimeDto StoreTime { get; set; }
        public StoreLocationDto StoreLocation { get; set; }
        public bool IsActive { get; set; }
        public byte Radius { get; set; }
    }
    public class TimeDto
    {
        public string? StartDate { get; set; }
        public string? EndDate { get; set; }
    }
    public class StoreLocationDto
    {
        public double Latitude { get; set; }
        public double LatitudeDelta { get; set; }
        public double Longitude { get; set; }
        public double LongitudeDelta { get; set; }
    }
    public class StoreFilterDto
    {
        public string searchValue { get; set; }
    }
   
}

File: ui/Components/Stores/Dtos/storeDto.ts
import { initialTimeDto, TimeDto } from "../../../Helpers/DataGrid/CrudTimeDto"

export interface StoreDto {
    id:number
    storeName: string
    storeTime: TimeDto,
    storeLocation: StoreLocationDto
    isActive:boolean
    radius:number
}

export const initialStoreLocationDto: StoreLocationDto = {
    latitude: 40.04144575513158,
    longitude: 32.908030908625896,
    latitudeDelta: 10.00,
    longitudeDelta: 10.00
}
export const initialStoreDto: StoreDto = {
    storeName: "",
    storeTime: initialTimeDto,
    storeLocation: initialStoreLocationDto,
    isActive: false,
    id: 0,
    radius: 0
}
export interface StoreLocationDto {
    latitude: number
    longitude: number,
    latitudeDelta: number,
    longitudeDelta: number
}

export interface StoreMarkerDto {
    latitude: number 
    longitude:number,
    latitudeDelta: number,
    longitudeDelta: number
}
export const  initialStoreMarkerDto:StoreMarkerDto = {
    latitudeDelta: 0,
    longitudeDelta: 0,
    latitude: 0,
    longitude: 0
}

export interface StoreFilterDto{
    searchValue:string
}
export const initialFilterDto:StoreFilterDto={
    searchValue: ""
}

export type HelperTextStoreDto={
    storeName:boolean
    latitude:boolean
    longitude:boolean
  }